Makefile

```diff
@@ -32,7 +32,12 @@ fmt:
 	@echo gofmt
 
 image:
-	podman build  -f Dockerfile.rhel8 -t $(LOCAL_IMAGE_TAG) .
+	buildah build --platform=$(TARGET_PLATFORMS) --manifest eventrouter -f Dockerfile.rhel8
+	buildah tag localhost/eventrouter $(IMAGE_REPOSITORY_NAME)
+
+image-push: image
+	podman manifest push --all $(IMAGE_REPOSITORY_NAME) docker://$(IMAGE_REPOSITORY_NAME)
```

Comment:
I'm unfamiliar here but why do we need the "docker" prefix given podman does not require a docker daemon?

Reply:
The destination can be a directory or other things and here we (at least I) want to push to an image in a registry implementing the Docker Registry HTTP API V2.